internal/js/modules/k6/browser/common/js/injected_script.js

```diff
@@ -14,6 +14,1259 @@
  * limitations under the License.
```

Comment:
The main implementation is in the `injected_script.js` file. The code was directly copied over from the compiled JS code from Playwright. I have added as many integration tests as possible to ensure that all the possible flows are accounted for. While i understand the jist of what is happening in the copied code, some of it is still a bit of a mystery as to why things are done this way. A lot of the code seems to be inherited behaviour from specifications, such as not being able to work with `header` or `footer` unless they're not encapsulated -- exact spec is unknown though.

Do we need to add the Playwright license and notice file in the browser module?

Reply:
I don't think we need to add another license note, as we already have this one:

https://github.com/grafana/k6/blob/113abf5fdd7bc5ddc36c87a10aa7febdf7a0bb71/internal/js/modules/k6/browser/common/js/injected_script.js#L1-L15